api/src/models/priceHistory.model.ts

```diff
@@ -0,0 +1,28 @@
+import { model, Schema, Document } from 'mongoose';
+import { PriceHistoryData } from '@interfaces/priceHistory.interface'; 
+
+const PriceHistorySchema: Schema = new Schema({
+// Will potentially calculate the date with BE logic
```

Comment:
You'll need this in the schema, regardless. My feedback here was in reference to the DTO.

Reply:
Sounds good. Just added those fields back into the code